### ISSUE TITLE
User getting multiple confirmation emails

### ISSUE THREAD

**Issue by aaronsamuel137** [phetsims/website#276]
@oliver-phet mentioned that one user keeps getting several confirmation emails per day. Here is my email response:

> According to the logs, it looks like he is indeed getting sent many confirmation emails a day. The only reason this would be happening is because someone must be making requests to his confirm email link over and over (the link that he got sent in his email to click to confirm his account). Not sure if he has this link open in a tab somewhere that keeps getting reloaded, or maybe some other entity like a search engine is somehow using this link.

> It doesn't appear to be happening to any other accounts as far as I can tell.

> I will look into adding a check in the website code so it doesn't send emails if you are already confirmed.

**Comment by aaronsamuel137** [comment]
I have manually fixed this users account by changing his confirmation code. His old confirm email link will now no long work, and probably what happened was that some search engine bot got ahold of his confirm email link and kept reconfirming it.

After looking into this some more, it appears that the reason we allow multiple confirmation emails to be sent is in case someone already registered tried to separately register for the newsletter (instead of going into their profile and checking receive email). In this case we want to send them another email so they can be assured that we got their request.

Considering this, we have 2 options here:

1. Leave it the way it is and assume this case was a rare fluke.
2. Spend more time looking into a more robust solution.
